<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraInfoDialog</class>
 <widget class="QDialog" name="CameraInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/context_actions/res/settings-small.png</normaloff>:/new/context_actions/res/settings-small.png</iconset>
  </property>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MJPEG stream URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio stream URL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSourceUrl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAudioSourceUrl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leSourceUrl</tabstop>
  <tabstop>leAudioSourceUrl</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbOK</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CameraInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOK</sender>
   <signal>clicked()</signal>
   <receiver>CameraInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CameraInfoDialog</receiver>
   <slot>checkEnteredData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSourceUrl</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CameraInfoDialog</receiver>
   <slot>checkEnteredData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkEnteredData()</slot>
 </slots>
</ui>
